### PR TITLE
Add whitelist to datafile size 

### PR DESCRIPTION
**Pull request by Morshedk**

### CODE REVIEW

File: src/SQLChecks/Tests/Databases.tests.ps1 [hunk 1]
```diff
@@ -33,11 +33,12 @@ Describe "No large fixed growth transaction logs" -Tag MaxTLogAutoGrowthInKB {
 
 Describe "Data file space used" -Tag MaxDataFileSize {
     $spaceUsedPercentLimit = $Config.MaxDataFileSize.SpaceUsedPercent
+    $WhiteListFiles =$Config.MaxDataFileSize.WhiteListFiles
```
Comment (taddison) [inline src/SQLChecks/Tests/Databases.tests.ps1:36]: We don't need this as it is extracted by the config in the function

```suggestion
```

File: src/SQLChecks/Tests/Databases.tests.ps1 [hunk 2]
```diff
@@ -33,11 +33,12 @@ Describe "No large fixed growth transaction logs" -Tag MaxTLogAutoGrowthInKB {
...
 
     $databases = Get-DatabasesToCheck @databasesToCheckParams
     foreach ($database in $databases) {
         It "$database files are all under $spaceUsedPercentLimit% full on $serverInstance" {
-            @(Get-DatabaseFilesOverMaxDataFileSpaceUsed -Config $Config -Database $database).Count | Should -Be 0
+            @(Get-DatabaseFilesOverMaxDataFileSpaceUsed -Config $Config -Database $database -WhitelistFiles $WhiteListFiles ).Count | Should -Be 0
```
Comment (taddison) [inline src/SQLChecks/Tests/Databases.tests.ps1:41]:
```suggestion
            @(Get-DatabaseFilesOverMaxDataFileSpaceUsed -Config $Config -Database $database).Count | Should -Be 0
```

**Review by taddison — CHANGES_REQUESTED** [review]
Suggestions added

**Comment by Morshedk** [comment]
unless I'm missing something, you're suggesting to leave it as it is?

**Comment by Morshedk** [comment]
because your suggestions are to revert the changes of the PR

**Comment by Morshedk** [comment]
Ah, I see how it works, Im going to cancel the PR

**Pull request closed without merging**